bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java

```diff
@@ -213,8 +213,12 @@ private String replaceKeysWithFunction(String template, Function<String, String>
         StringBuffer sb = new StringBuffer();
 
         while (m.find()) {
-            String key = m.group(1);
-            m.appendReplacement(sb, getMessage.apply(key));
+            try {
+                String key = m.group(1);
+                m.appendReplacement(sb, getMessage.apply(key));
+            } catch (Exception e) {
+                logger.debug("Error occurred during template filling, cause ", e);
```

Comment:
Now, if error occurs, method return tile which name or link url is not valid. User can still see tile from dashboard and recognise the problem. If method return empty string, invalid tile is not added at all. Which one is better? Or any other suggestions?

Reply:
No strong opinion, showing it broken should be alright.